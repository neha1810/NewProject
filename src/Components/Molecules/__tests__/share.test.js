import React from 'react';
import {shallow} from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Share from '../share/share';

configure({ adapter: new Adapter() });

describe('Share', () => {
let wrapper;
beforeEach(()=> {
wrapper = shallow(
        <Share />
    )
})


it('render the Share component', () => {
    expect(wrapper).toMatchSnapshot();
});
// it('Button exists',()=> {
//     //wrapper.find('button').at(1).simulate('click');
//     expect(wrapper.find('#Share')).toBeDefined();
// })



})