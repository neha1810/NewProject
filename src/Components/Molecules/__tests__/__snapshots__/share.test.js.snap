// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Share render the Share component 1`] = `ShallowWrapper {}`;
